Watchman.Discord/Areas/Help/Controllers/HelpController.cs

```diff
@@ -17,30 +20,53 @@ public class HelpController : IController
+        }
 
+        private Task PrintHelpForAllCommands(HelpCommand command, Contexts contexts, IEnumerable<HelpInformation> helpInformations)
+        {
+            var messagesService = this._messagesServiceFactory.Create(contexts);
             if (command.Json)
             {
                 var helpMessage = this._helpMessageGenerator.GenerateJsonHelp(helpInformations);
-                await messagesService.SendMessage(helpMessage, MessageType.Json);
+                return messagesService.SendMessage(helpMessage, MessageType.Json);
             }
-            else
+            var availableCommandsResponse = this._responsesService.GetResponse(contexts.Server.Id, x => x.AvailableCommands());
+            var hereYouCanFindAvailableCommandsWithDescriptions = this._responsesService.GetResponse(contexts.Server.Id, x => x.HereYouCanFindAvailableCommandsWithDescriptions());
+            return messagesService.SendEmbedMessage(title: availableCommandsResponse, description: hereYouCanFindAvailableCommandsWithDescriptions,
+                this._helpMessageGenerator.MapHelpForAllCommandsToEmbed(helpInformations, contexts.Server));
+        }
+
+        private Task PrintHelpForOneCommand(HelpCommand command, Contexts contexts, IEnumerable<HelpInformation> helpInformations)
+        {
+            string normalizeCommandName(string x) => x.ToLowerInvariant().TrimStart('-').Replace("command", "");
+            var helpInformation = helpInformations.FirstOrDefault(x => normalizeCommandName(x.CommandName) == normalizeCommandName(command.Command));
+            if (helpInformation == null)
             {
-                await messagesService.SendEmbedMessage("Dostępne komendy:",
-                    "Poniżej znajdziesz listę dostępnych komend wraz z ich opisami\nJeśli chcesz poznać dokładniejszy opis - zapraszamy do opisu w naszej dokumentacji\nhttps://watchman.readthedocs.io/pl/latest/156-lista-funkcjonalnosci/",
-                    this._helpMessageGenerator.MapToEmbedInput(helpInformations)); //todo add to responses - now we cannot handle this format
+                return Task.CompletedTask;
             }
+            var messagesService = this._messagesServiceFactory.Create(contexts);
+            var helpMessage = this._helpMessageGenerator.MapHelpForOneCommandToEmbed(helpInformation, contexts.Server);
+            var noDefaultDescriptionResponse = this._responsesService.GetResponse(contexts.Server.Id, x => x.NoDefaultDescription());
+            var description = helpInformation.Descriptions
+                .FirstOrDefault(x => x.Language == helpInformation.DefaultLanguage)?.Text ?? noDefaultDescriptionResponse;
+            var howToUseCommand = this._responsesService.GetResponse(contexts.Server.Id, x => x.HowToUseCommand());
+            return messagesService.SendEmbedMessage($"{howToUseCommand} {helpInformation.CommandName}", description, helpMessage);
```

Comment:
może jakiś : między `{howToUseCommand} {helpInformation.CommandName}`

Reply:
właśnie lepiej wygląda bez
wtedy to jest tak:
`Jak używać komendy Mute`